packages/shared/providers.yaml

```diff
@@ -6106,6 +6107,7 @@ workable:
             pattern: '^[a-z0-9_-]+$'
             example: domain
             suffix: .workable.com
+            optional: true
```

Comment:
I went this route thinking it's better to let someone just hard code the subdomain if they know it, given some of the other complexity we saw with this change. Also the validation fails if you have use a non-specified `connection_config` variable in the `base_url` field.

Reply:
For the validation, you can also improve it if it's in the way 👍🏻